src/test/java/de/wejago/mqtt2influx/config/MqttClientConfigTest.java

```diff
@@ -37,7 +37,7 @@ void testSetMqttClientSuccess() {
         // THEN
         assertThat(mqttClient).isNotNull();
         assertThat(mqttClient.getServerURI()).isEqualTo(BROKER);
-        assertThat(mqttClient.getClientId()).isEqualTo("mqtt-2-influx-db-application");
+        assertThat(mqttClient.getClientId()).isEqualTo("mqtt-2-influx-db-application-localhost");
```

Comment:
You can use `startsWith` instead of `isEqualTo` otherwise the test could fail if the IP resolution doesn't work as expected.